jobs/JOBSPROC_RAP_DUMP

```diff
@@ -152,51 +154,51 @@ export SCRIPTSobsproc=${SCRIPTSobsproc:\
 # ---------------------------------------
 # Paths for dump and bufr_dumplist items
 # ---------------------------------------
-export HOMEbufr_util=${HOMEbufr_util:\
--${NWROOT:?}/bufr_util.${bufr_util_ver:?\
-"###FATAL ERROR \"bufr_util_ver\" is not set"}}
+export HOMEbufr_dump=${HOMEbufr_dump:\
+-${PACKAGEROOT:?}/bufr_dump.${bufr_dump_ver:?\
+"###FATAL ERROR \"bufr_dump_ver\" is not set"}}
```

Comment:
These 3 lines are redundant and not required. $HOMEbufr_dump is already predefined when the bufr_dump/1.0.0 module was loaded. I suggest removing these lines.